examples/deeplab/README.md

```diff
@@ -0,0 +1,37 @@
+# Examples of DeepLab
+
+## Performance
+DeepLab V3+
+
+| Network backborn | Training | Evaluation | Eval scales | Reference | ChainerCV (weight conversion) |
+|:-:|:-:|:-:|:-:|:-:|:-:|
+| Xception65 | VOC2012 trainaug | VOC2012 val | (1.0,) |  82.36 % * |  82.36 % |
```

Comment:
In my environment, `| (1.0,) |` is not rendered properly (the space may not be a space)

Reply:
sorry for my careless check...